backend/app/config.py

```diff
@@ -0,0 +1,10 @@
+from pydantic import BaseSettings, SecretStr
+
+
+class Settings(BaseSettings):
+    api_token: SecretStr
+
+    max_ws_connections: int = 2
```

Comment:
Should this be configurable? I mean, is it possible to have a situation when we want this to be any number other than 2?

Reply:
What about the situation when an user has several tabs of a app: the user decided to open a new window, but did't close the old one, in these keys the user should get messages from in both

Reply:
Wow, that is a very good point!

I think, this use-case is a nice thing to support. However, I then don't understand the point of the `max_ws_connections` constant. Does it limit the number of tabs one user can connect with? And what behavior is expected if a user opens more tabs than we want them to?